editor/cypress/fixtures/titanic.json

```diff
@@ -9,12 +9,32 @@
       "@id": "ml:data",
       "@type": "@json"
     },
-    "dataBiases": "ml:dataBiases",
-    "dataCollection": "ml:dataCollection",
+    "dataAnnotationAnalysis": "cr:dataAnnotationAnalysis",
```

Comment:
We should introduce a separate namespace (and prefix) for the RAI extension, e.g.,

"rai": "https://mlcommons.org/croissant/RAI/"

Reply:
As a side note, it's not strictly necessary to add all these attributes to the json-ld context... It's not that bad to use them with a prefix, e.g., "rai:dataAnnotationDemographics". WDYT?

Reply:
We keep the RAI namespace in the context as:

"rai": "http://mlcommons.org/croissant-RAI/1.0/",

And the fields are now:

rai:dataBiases:[...]